app/src/ui/preferences/advanced.tsx

```diff
@@ -200,6 +200,11 @@ export class Advanced extends React.Component<
   }
 
   private renderNotificationsSettings() {
+    if (__LINUX__) {
+      // notifications not currently supported
+      return null
+    }
+
```

Comment:
Thank you for this! But I think this specific function should still be rendered, no? Notifications using `desktop-notifications` (which is what you disabled in `initializeDesktopNotifications`) shouldn't work, but I'd hope that notifications using HTML5 API should still work 🤔 

An easy way to test this is using Help -> Show Notification in a dev/test build

Reply:
> An easy way to test this is using Help -> Show Notification in a dev/test build

Let me test that and report back

Reply:
Yay!! 🎉